src/Dialect/Mlir/DialectBuilder.cpp

```diff
@@ -112,8 +112,18 @@ Value MathBuilder::xori(Value lhs, Value rhs) const {
 
 Value MathBuilder::add(Value lhs, Value rhs) const {
```

Comment:
are the changes to MathBuilder::add/mul/cast necessary for MatMulInteger? please consider adding an explanation of them in the PR description